packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -92,16 +92,27 @@ const WardViewByLocation = () => {
       return <WardBed key={bed.uuid} bed={bed} wardPatients={wardPatients} />;
     });
 
+    const patientsInBedsUuids = bedLayouts.flatMap((bedLayout) => bedLayout.patients.map((patient) => patient.uuid));
```

Comment:
I assume bedLayouts is always non-null?

Reply:
I tested on a location with no beds, and this worked, but it's a good, bigger point that now that I've removed the check to render nothing if bed management isn't installed we probably need a lot more checks around this, let me take a look.